From c433b09125678909bae8ae05ac0748773eb6f815 Mon Sep 17 00:00:00 2001
From: Yong Yao <yong.yao@intel.com>
Date: Tue, 2 Apr 2013 00:21:26 +0800
Subject: [PDK MR2 Temp] temp fix to pass the build

BZ: 117199

	add one MACRO -- KEYSTORE_MESSAGE_SIZE
	need be checked by related FT

Change-Id: I31f6d4f0877d49b3434fe635a6c2ca2be4e505ec
Signed-off-by: Yong Yao <yong.yao@intel.com>
---
 keystore/include/keystore/keystore_get.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/keystore/include/keystore/keystore_get.h b/keystore/include/keystore/keystore_get.h
index 4bddd70..b5fe657 100644
--- a/keystore/include/keystore/keystore_get.h
+++ b/keystore/include/keystore/keystore_get.h
@@ -18,6 +18,7 @@
 #define __KEYSTORE_GET_H__
 
 #include <sys/types.h>
+#define KEYSTORE_MESSAGE_SIZE 65535
 
 #ifdef __cplusplus
 extern "C" {
-- 
1.7.4.1

